package.json

```diff
@@ -75,7 +75,7 @@
     "js-test-jquery": "cross-env JQUERY=true npm run js-test-karma",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel js-lint css-lint lockfile-lint",
     "docs": "npm-run-all docs-build docs-lint",
-    "docs-build": "hugo --cleanDestinationDir --printUnusedTemplates",
+    "docs-build": "PATH=$(npm config get prefix)/bin:$PATH hugo --cleanDestinationDir --printUnusedTemplates",
```

Comment:
I had to change the path because, otherwise, `hugo` (via the npm command) will use the sass binary installed from our `package.json` that doesn't support `sass --embedded` -> `sass --embedded is unavailable in pure JS mode.`.
Following https://gohugo.io/functions/resources/tocss/#dart-sass, I've installed on my computer `sass` that supports `sass --embedded` so the script must use this version instead of the one in `node_modules/.bin`.